tests/test_abandoned_match_scan_268.py

```
@@ -0,0 +1,672 @@
+    reached the spawn would start a verifier nobody wants, on exactly the storage
+    where it may outlive SIGKILL. Guarded in the shared seam so every capture
+    caller gets it rather than the ones that remember to ask.
+    """
+    runner = runner_mod.SubprocessRunner(owner="test")
+    cancel = asyncio.Event()
+    cancel.set()
+
+    spawned = []
+    real_exec = runner_mod.asyncio.create_subprocess_exec
+
+    async def watched_exec(*args, **kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `watched_exec`

(ANN202)